core/matching/field_matcher.go

```diff
@@ -1,72 +1,41 @@
 package matching
 
 import (
-	"bytes"
-	"regexp"
-
-	"github.com/ChrisTrenkamp/goxpath"
-	"github.com/ChrisTrenkamp/goxpath/tree/xmltree"
-	"github.com/NodePrime/jsonpath"
 	"github.com/SpectoLabs/hoverfly/core/models"
-	glob "github.com/ryanuber/go-glob"
 )
 
 func FieldMatcher(field *models.RequestFieldMatchers, toMatch string) bool {
+	matches := []bool{}
```

Comment:
You can get rid of this, and then just return `false` whenever there isn't a match, otherwise return `true`